teamcity/resource_build_trigger_schedule.go

```diff
@@ -65,6 +66,44 @@ func resourceBuildTriggerSchedule() *schema.Resource {
 				Elem:     &schema.Schema{Type: schema.TypeString},
 				Optional: true,
 			},
+			"cron_schedule": {
+				Type:     schema.TypeList,
+				Optional: true,
+				MaxItems: 1,
+				Computed: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"seconds": {
+							Type:     schema.TypeString,
+							Optional: true,
```

Comment:
To confirm, if one of these is left out, is the behavior identical to having `*`?

Reply:
At this point no, but I think `*` would be a good value to have as a default.

Reply:
Another reasonable option IMO is to make every field required. This is also the case with cron itself and the explicitness could help in avoiding bugs related to having the wrong idea about default behavior (e.g. sometime assuming the default is 0).

Reply:
I like this idea, having every field required makes a user have to think about all the fields, and unable to rely on any defaults. Done, it'll be included in the next patchset.